src/module/MintFeeManagerModule.sol

```diff
@@ -0,0 +1,153 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.20;
+
+import {Module} from "../Module.sol";
+import {Role} from "../Role.sol";
+
+library MintFeeManagerStorage {
+
+    /// @custom:storage-location erc7201:mint.fee.manager
+    bytes32 public constant MINT_FEE_MANAGER_STORAGE_POSITION =
+        keccak256(abi.encode(uint256(keccak256("mint.fee.manager")) - 1)) & ~bytes32(uint256(0xff));
+
+    struct Data {
+        address platformFeeRecipient;
+        uint256 defaultMintFee;
+        mapping(address => uint256) mintFees;
```

Comment:
provide state names for the k/v in the `mapping`

Reply:
addressed